### PR TITLE
Add toolbar icon style and set for relevant image views.

### PR DESCRIPTION
**Pull request by hugokallstrom**
### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

**Comment by hedvigoscar** [comment]
Which tickets does this PR pertain to? 😉

**Comment by hugokallstrom** [comment]
> Which tickets does this PR pertain to? 😉

Sorry, forgot to include jira number! https://hedvig.atlassian.net/browse/HVG-812

**Comment by hedvigoscar** [comment]
HVG-812

**Comment by hedvigoscar** [comment]
tried to get them to associate but no dice :grimac

File: app/src/main/res/values/styles.xml
Comment (hedvigoscar) [inline app/src/main/res/values/styles.xml:123]:
```suggestion
        <item name="android:background">?selectableItemBackgroundBorderless</item>
```

**Review by hedvigoscar — APPROVED** [review]
Approved after merging suggestion 🙂

**Pull request merged**
